tests/test_browser_model.py

```diff
@@ -12,18 +12,18 @@ def test_new_private_window(self, selenium):
         """Tests opening a new private browsing window via menu"""
         foxpuppet = FoxPuppet(selenium)
         foxpuppet.browser.open_window(private=True)
-        assert len(foxpuppet.browser.windows.all) == 2
+        assert len(foxpuppet.windows.all) == 2
         assert foxpuppet.browser.is_private is False
-        foxpuppet.browser.windows.focus(foxpuppet.browser.windows.all[1])
+        foxpuppet.windows.focus(foxpuppet.windows.all[1])
```

Comment:
I would say that `foxpuppet.windows` should return a collection of window objects. This should therefore be `foxpuppet.windows[1].focus` and line 15 should be `len(foxpuppet.windows)`.